### PR TITLE
Add config example for SOCIETATIS

### PR DESCRIPTION
**Pull request by tommR1**
Adds config example for Societatis [SCTS] https://societatis.io/

“an organization formed for a particular purpose“

### CODE REVIEW

File: config_examples/societatis.json [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+			"stats": "/stats",
+			"enable": "/enable",
+			"disable": "/disable"
+		}
+	},
+
+	"monitoring": {
+		"daemon": {
+			"checkInterval": 60,
+			"rpcMethod": "getblockcount"
```
Comment (BlackPeter13) [inline config_examples/societatis.json:247]: "rpcMethod": "get_blockcount"

File: config_examples/societatis.json [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+		},
+		"wallet": {
+			"checkInterval": 60,
+			"rpcMethod": "getbalance"
```
Comment (BlackPeter13) [inline config_examples/societatis.json:251]: "rpcMethod": "get_balance"

File: config_examples/societatis.json (hunk 1 above)
Comment (tommR1) [reply config_examples/societatis.json:247]: Hi thanks. But no, we use a different notation than Monero. That fits.
`getblockcount`

File: config_examples/societatis.json (hunk 2 above)
Comment (tommR1) [reply config_examples/societatis.json:251]: `getbalance` is also the right method.

**Review by tommR1 — COMMENTED** [review]
ACK

**Review by dvandal — APPROVED** [review]

**Review by mahdiamer — COMMENTED** [review]
Good

**Pull request merged**